pkg/planner/core/rule_join_reorder.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/pingcap/tidb/pkg/expression"
 	"github.com/pingcap/tidb/pkg/parser/ast"
+	base2 "github.com/pingcap/tidb/pkg/planner/core/base"
```

Comment:
Can I ask why do we need to rename here?

Reply:
there is another core pkg in the import list here, oh! for this file, its not, I will change it back to base

Reply:
nice catch